lib/Dropdown/DropdownAction.js

```diff
@@ -20,9 +20,16 @@ const DropdownAction = ({
   ...props
 }) => {
   const {
-    toggleShowContent = props.toggleShowContent,
-    showContent = props.showContent
+    showContent: showContentProp,
+    toggleShowContent: toggleShowContentProp,
+    ...restProps
+  } = props;
```

Comment:
same q for the one below, why are we destructuring again here rather than in the main props destructuring above?

Reply:
in this case there was a naming collision when I tried to destructing in the function arguments directly with the destructuring that happens on the DropdownContext below.

I guess i could rename the values in the component props when destructuring like this but it didn't feel right somehow 

```
...
  value,
  showContent: showContentProp,
  toggleShowContent: toggleShowContentProp,
  ...props
}) => {
  const {
    toggleShowContent = toggleShowContentProp,
    showContent = showContentProp
  } = useContext(DropdownContext) || {};
...
```


I guess my actual issue is refactoring this bit to prevent the collision:
```jsx
  const {
    toggleShowContent = toggleShowContentProp,
    showContent = showContentProp
  } = useContext(DropdownContext) || {};
```

I'll have a rethink - think it was my late afternoon brain failing me on this one

Reply:
Ah, the other issue was alongside the naming issue, we are also setting default values so that is the dropdown context returns nothing, we default to the values from the props - that tripped up my other attempts too